packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-development.md

```diff
@@ -1,116 +1,125 @@
 ---
-id: Product development guide
+id: Develop accessibly
 section: accessibility
 ---
 import { AccessibilityDatalist } from './accessibility-datalist.js';
 
-## General questions to consider
+## Prioritizing accessibility during development
 
-To assess how accessible your applications or products are, consider these questions:
-- **Is it discoverable/perceivable by all users?** If you can see or click on it, then it should be discoverable and perceivable by all users. You should determine if you can easily navigate to it by keyboard or by screen reader.
-- **Is it interactable?** If you can get to it, it should be easy to actually use once you’ve focused on the element. You should be able to interact with it by keyboard (for example, if it’s a button, can you press **Enter** to initiate the action?). Also use a screen reader to ensure you can initiate the action.
-- **Is it understandable?** If the element can be found and interacted with, it should be clear what this action does. If it’s a button, you should determine if it has visible text that would be clear out of context of the page. If it doesn’t have visible text, you should check if it has an aria-label or accessible name.
+As you develop your product, it can be helpful to keep accessibility top of mind, by asking yourself a few questions: 
 
+**1. Can all users discover and perceive elements?**
 
-## What products should address
+If you can see or click on it, then it should be discoverable and perceivable by all users, including those using a keyboard or screen reader.
 
-Keep in mind that while PatternFly provides accessible components, we can't guarantee that your products will be accessible. There are a few items outside the scope of PatternFly, so you should keep these items in mind to ensure accessibility in anything you build: 
-<br />
+**2. Can all users interact with elements?**
 
+It should be easy for users to use an element once it's in focus. They should be able to interact with elements and initiate actions via keyboard (for example, pressing **Enter** to initiate a button action) and screen readers.
 
-<AccessibilityDatalist />
+**3. Can all users understand elements?**
+
+It should be clear what actions elements can perform. For example, buttons should have visible text that would be clear out of context of the page. If not, it should have an aria-label or accessible name.
+
+## Extra steps for accessibility 
```

Comment:
I have some concerns about this title, only because in the past I frequently faced the belief that using a PF component would automatically mean that a product's UI is 100% accessible. Calling it "Extra steps" makes me worried that people will skim over that section, assuming that was if they wanted to do extra credit so to speak, which most developers won't feel like they have time to do. I'd rather title this something that doesn't make it sound not necessary, you know? I'm terrible at naming so I worry to give different suggestions, but I can try if it'd be helpful. 😄

Reply:
What about something like "Considerations to take as a developer"